src/containers/VisitPage.js

```diff
@@ -360,17 +335,28 @@ class VisitPage extends Component {
             minWidth='874.5'
             style={{ padding: '14px 0' }}>
             <StepBar
-              currentStepIndex={ this.state.currentStep }
-              steps={ this.steps }
+              currentStepKey={ this.props.stepKey }
```

Comment:
From what I can tell, this could be `step.key` here, which might be more consistent with how it's being used further down.

Reply:
Non-blocking, @todo